lib/clients/AzureClient.py

```diff
@@ -143,8 +175,9 @@ def _create_snapshot(self, volume_id):
         try:
             disk_info = self.compute_client.disks.get(
                 self.resource_group, volume_id)
-            snapshot_name = '{}-{}'.format(self.snapshot_prefix,
-                                           time.strftime("%Y%m%d%H%M%S"))
+            snapshot_name = '{}-{}-{}'.format(self.snapshot_prefix,
```

Comment:
can extract this into a function and reuse it in 3 places  : 

`
generate_name(prefix,self)
{
return 
'{}-{}-{}'.format(prefix, time.strftime("%Y%m%d%H%M%S"))                                           random.randrange(10000, 99999),time.strftime("%Y%m%d%H%M%S"))
}
`

Reply:
Done